### ISSUE TITLE
Hi, I will explain this problem.

### ISSUE THREAD

**Issue by Sraw** [pe3zx/huawei-block-list#3]
First, I understand that you may not know what is "beian".

It is one kind of censorship in China which is used to prevent illegal websites such as a sexsiual website or gaming website. Every website(running on port 80 or 443) in China(with `.cn` domain or server is running in China) needs to have a record on "beian" system. Otherwise, the domain cannot be resolved to that IP address.

So theoretically you would see this issue on every Chinese website that includes "beian" information on their pages no matter what device you are using. Well, I know that's shitty... But admittedly every government has its own way to censor the internet. The Chinese government is just too naive to do it better.

To summarize, it is not a device specific issue. But whatever device you are using to access a Chinese website, you may meet this issue. Even you are using an iPhone or Pixel.

And it is impossible to really protect yourself(if you really need to) by only blocking that particular domain. As you cannot trust any Chinese website, they may share your access records with the government. I would suggest using a VPN, that is the best way to protect yourself :)

**Comment by pe3zx** [comment]
Hi @Sraw,

Thank you for your useful information. I will verify this again and add to `README. md`.

I also realized after analyzing `baidu. com` that this claim isn't device specific issue resulting as described in `analysis.md`. 

Before I initiated this repository, the only record I added to the block list is `*. cn`.
